sds_data_manager/lambda_code/SDSCode/upload_api.py

```diff
@@ -50,25 +50,30 @@ def lambda_handler(event, context):
 
     :return: A pre-signed url where users can upload a data file to the SDS.
     """
-    logger.info(f"Event: {event}")
```

Comment:
I found event logs helpful when looking through logs.

Reply:
It is included down below now in a debug log.
`logger.debug("Parsing path parameters=[%s] from event=" "[%s]", path_params, event)`

Reply:
ah yes. Didn't see that